attestation-agent/kbs_protocol/protos/attestation-agent.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package attestation_agent;
+
+message GetEvidenceRequest {
+    bytes RuntimeData = 1;
+}
+
+message GetEvidenceResponse {
+    bytes Evidence = 1;
+}
+
+message GetTokenRequest {
+    string TokenType = 1;
+}
+
+message GetTokenResponse {
+    bytes Token = 1;
+}
+
+service AttestationAgentService {
+    rpc GetEvidence(GetEvidenceRequest) returns (GetEvidenceResponse) {};
```

Comment:
I added this the two API for the attestation-agent in future. Please check if this is good to you @mkulke

Reply:
No. There will be a http proxy that translate ttrpc to RESTful API to expose to workload containers. THe routine will be
`cdh` --> `proxy` --> `app container`  or
`aa` --> `proxy` --> `app container`

We've talked about this in the slack https://cloud-native.slack.com/archives/C039JSH0807/p1690523890291199